mlem/contrib/flyio/utils.py

```diff
@@ -10,9 +11,20 @@
 FLY_TOML = "fly.toml"
 
 
+logger = logging.getLogger(__name__)
+
+
 def check_flyctl_exec():
     try:
-        run_flyctl("version", wrap_error=False)
+        cmd = run_flyctl("version --json", wrap_error=False)
+        output = json.loads(cmd.decode())
+        version = output.get("Version", None)
+        if not version or version.split(".")[1] != "1":
```

Comment:
recommend using [semver](https://pypi.org/project/semver/) to parse and compare versions
here it would be:
```python
version_str = output.get("Version", None)
if version_str is not None:
  version = semver.Version.parse(version_str)
  if ver.major < 1:
     ... 
```